.github/workflows/build.yml

```
@@ -113,8 +164,18 @@ jobs:
           path: build/
           key: ${{ matrix.artifact_name }}-v${{ env.ONNXRUNTIME_VERSION }}-cache-v1-${{ hashFiles('matrix.json') }}
 
+      - name: Install tree
+        if: steps.cache-build-result.outputs.cache-hit != 'true' && runner.os == 'Windows'
+        uses: msys2/setup-msys2@v2
+        with:
+          install: tree
+
+      - name: Install tree

```

treeのインストールは確認用にですよね。
必須ではないことがわかるようにコメントを書いておくとあとあとファクタリングとかしやすいのかなと思いました！
```suggestion
      - name: Install tree for debug
```